apps/frontend/src/libs/components/popup-warning/libs/castome-hook/use-geolocation.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+
+const useGeolocation = () => {
+	const [geolocation, setGeolocation] = useState<boolean>(false);
```

Comment:
Can we rename it to something more logical for booleans
```suggestion
	const [isGeolocationEnabled, setIsGeolocationEnabled] = useState<boolean>(false);
```